README.md

```diff
@@ -9,14 +9,16 @@ Development
 ### Prerequisites
 
 We highly recommend using VSCode for development, if you don't have it, make sure you get it installed. You will also need to install:
-`homebrew`, `docker`, `npm`.
+`homebrew`, `docker`, `npm`. You should also install the package `dbaeumer.vscode-eslint` for syntax linting & format errors in VS Code.
 
 ### Running locally
 
 Make sure that you clone this repo and are in the `ui` folder. then simply do:
 
-    npm install
-    npm start
+    make install
+    make run
+
```

Comment:
after merge, make sure you tell people that they should use make install and make run, or they might be confused why the starting of the ui suddenly started failing